### ISSUE TITLE
Can this repo be archived?

### ISSUE THREAD

**Issue by nanometrenat** [IFRCGo/cbs#1224]
Hi there, I believe that the CBS platform that this repo's code relates to has now been superceded by [Nyss](https://cbs.ifrc.org/what-nyss), is that correct?

Should we therefore update the readme to instead direct people towards the Nyss repo (https://github.com/nyss-platform-norcross/nyss) and archive this "cbs" repo?

**Comment by alexanderhoset** [comment]
This seems correct. 

@gardnk do you have the rights to archive this repo?

**Comment by gardnk** [comment]
I agree, and I have the rights. I added a link to the nyss repo in PR #1225
When that is merged, I can archive this repo.

**Comment by nanometrenat** [comment]
Thank you very much @gardnk and @alexanderhoset !  I have approved/merged the PR so over to you @gardnk to archive if happy.

FYI re the other [Nyss repo](https://github.com/nyss-platform-norcross/nyss) I see that the about section on the right still links to [cbsrc.org](https://cbsrc.org/) - might be worth updating that to the new link as well, if you have the powers.

Thanks again

**Comment by gardnk** [comment]
Good catch @nanometrenat! I no longer work with the nyss team, but I did manage to update the link 🙂
